services/trex/core/server/auth/authz.ts

```diff
@@ -283,6 +287,14 @@ export async function authz(c: Context, next: any) {
         }
       }
     }
+    if (!bearerToken && c.req.url) {
```

Comment:
auth token is attached to the params in enterprise gateway (based on their library)